### PR TITLE
Added kinematics method to arms class

### PR DESCRIPTION
**Pull request by devin1234-droid**
Ask Ben for feedbacks for values of As and Ae.

### CODE REVIEW

**Comment by bznerd** [comment]
@lance [launch.json](launch.json) should be left alone since we don't use VS Code to directly run or debug code right?

File: src/main/java/frc/robot/subsystems/Arm.java [hunk 1]
```diff
@@ -15,6 +15,8 @@ public class Arm extends SubsystemBase {
       new CANSparkMax(CanId.shoulderNeo2, MotorType.kBrushless);
   private final CANSparkMax elbowNeo = new CANSparkMax(CanId.elbowNeo, MotorType.kBrushless);
   private final WPI_TalonSRX leftLead = new WPI_TalonSRX(CanId.turret);
+  private final double As = 0.0;
```
Comment (bznerd) [inline src/main/java/frc/robot/subsystems/Arm.java:18]: These should be the Lp/Lf (length proximal and forearm variables) and the method parameters As Ae. Also maybe rename those theta_shoulder or smth so it's a bit more readable.

Lf and Lp should also go in Constants.java, but all the arm/gripper constants need this, so I'll open an issue for all of them to be resolved in a separate PR.

Good work :+1:

**Comment by lance** [comment]
> @lance [launch.json](launch.json) should be left alone since we don't use VS Code to directly run or debug code right?

Yeah - not sure how that got into the commit. I'm still a Code Spaces newb

**Review by lance — APPROVED** [review]

**Review by bznerd — APPROVED** [review]
Next up inverse kinematics :+1:

**Pull request merged**